addon/mixins/touch-action.js

```diff
@@ -37,15 +36,15 @@ export default Mixin.create({
   ignoreTouchAction: false,
 
   init() {
-    this._super();
+    this._super(...arguments);
 
     const {
       tagName,
       ignoreTouchAction,
       click
     } = this;
 
-    const hasClick = click !== K;
+    const hasClick = typeof click === 'function';
```

Comment:
`click !== function() { return this; } ` vs `typeof click !== 'function` - but ur question is why cant we just check for the property itself, without caring about type?